lib/ChangeDueDateDialog/DueDatePickerForm.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form';
+import { Row, Col } from '@folio/stripes-components/lib/LayoutGrid';
+import Datepicker from '@folio/stripes-components/lib/Datepicker/Datepicker';
+import Timepicker from '@folio/stripes-components/lib/Timepicker/Timepicker';
+
+class DueDatePickerForm extends React.Component {
+  static propTypes = {
+    dateProps: PropTypes.object,
+    timeProps: PropTypes.object,
+  }
+
+  render() {
+    const { dateProps, timeProps } = this.props;
+
+    return (
+      <Row>
+        <Col xs={12} sm={6} md={3}>
+          <Field
+            name="date"
+            component={Datepicker}
+            ignoreLocalOffset
+            {...dateProps}
+          />
+        </Col>
+        <Col xs={12} sm={6} md={3}>
+          <Field
+            name="time"
+            component={Timepicker}
+            timezone="UTC"
+            {...timeProps}
+          />
+        </Col>
+      </Row>
+    );
+  }
+}
+
+export default reduxForm({
```

Comment:
@doytch do we care about showing the "unsaved changes" popup in cases when somebody will want to leave the due date dialog before saving? If no then this is fine. If we do then we may need to replace this with `stripes-form`.

Reply:
Good catch. For whatever reason it never throws the "unsaved changes" popup for me anyway. 🤔 

Will change it anyway though! :D

Reply:
Cool. I think the `navigationCheck` has to be set to `true`.